### PR TITLE
add [NODERS]TEAM endpoints

### PR DESCRIPTION
**Pull request by nodersteam**

### CODE REVIEW

File: archway/chain.json [hunk 1]
```diff
@@ -265,6 +273,10 @@
         "address": "grpc.mainnet.archway.io:443",
         "provider": "Archway Foundation"
       },
+      {
+        "address": "http://archway.grpc.nodersteam.com:9090",
```
Comment (kayano) [inline archway/chain.json:277]:
```suggestion
        "address": "archway.grpc.nodersteam.com:9090",
```

Comment (jlehtimaki) [inline archway/chain.json:277]: Adress cannot have `http://` in it
```suggestion
        "address": "archway.grpc.nodersteam.com:9090",
```

**Review by jlehtimaki — CHANGES_REQUESTED** [review]

File: archway/chain.json (hunk 1 above)
Comment (nodersteam) [reply archway/chain.json:277]: Sorry, my mistake

**Comment by kayano** [comment]
@nodersteam please resolve conflicts

**Pull request closed without merging**